### ISSUE TITLE
Quick pick: tab should focus action, not close the quick pick

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51850]
**Environment Details:**
VS Code Version:1.24
OS Version:Windows10
**Additional Details:**
Keyboard,MAS3.2.1
**Steps to Reproduce:**
1.Launch VS Code.
2.Navigate to Go(Alt+G) menu option and select "Go To File" from dropdown.
3.Use the "Tab key" to navigate  through the tree view.
**Actual:**
On pressing the "tab" key "Type "?" to get help from the actions you can get from here" tree view closes.
**Expected:**
On pressing the "tab" key "Type "?" to get help from the actions you can get from here"  tree view should not close.
**"Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**[User] Impact:**
User will not be able to use tree view.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
![accessibility a11y_vscode_gotofile_keyboardgotofile](https://user-images.githubusercontent.com/39204909/41397430-cd638258-6fd1-11e8-9221-641e1c3ae392.png)
[vedioShowOenEditors.pptx](https://github.com/Microsoft/vscode/files/2101682/vedioShowOenEditors.pptx)

**Comment by isidorn** [comment]
This issue has a lot of duplicates, let's assign it to july to discuss what is needed for this
ping @bpasero

**Comment by bpasero** [comment]
Hitting `Tab` once should just move focus into the tree, that works for me. We cannot navigate in the tree via `Tab` key because the items do not have `tabindexes`.

**Comment by isidorn** [comment]
Yeah that makes sense. Also note this issue https://github.com/Microsoft/vscode/issues/52000
Which leans towards that the focus should always be on the first tree item, not on the whole tree

**Comment by bpasero** [comment]
Why? You can have a tree focused but not any element.

**Comment by isidorn** [comment]
@bpasero i understand the logic behind that. But why would I as a user ever want the tree to be focused - what value does that give me?

**Comment by bpasero** [comment]
@isidorn just to give a concrete example: the context menu on the explorer tree allows to use commands on a folder by right clicking into empty space. if the first item was always selected, how would that be possible otherwise?

**Comment by JacksonKearl** [comment]
@bpasero How are you getting `tab` to move focus to the quick pick items? For me it always closes the quick pick, except for `cmd-p` quick pick, where `tab` moves focus to the `open to the side` icon.

**Comment by bpasero** [comment]
@JacksonKearl you are right, it only seems to work because that particular quick pick has an action in the items.

**Comment by JacksonKearl** [comment]
@bpasero so do we keep current behavior or make tab move focus to the results list?

**Comment by bpasero** [comment]
I think it would make sense that hitting Tab once from quick open moves focus to the list/tree control if possible.
